.travis.yml

```diff
@@ -50,6 +50,7 @@ before_deploy:
   - export PRERELEASE=false
   - if $(echo $TRAVIS_BRANCH | grep -Eq "v([0-9]+.[0-9]{2}.[0-9]+)-rc.[0-9]+"); then export PRERELEASE=true; fi
   - echo "Deploying $TRAVIS_BRANCH, prerelease $PRERELEASE"
+  - git submodule update --init --recursive
```

Comment:
It looks like the git submodule reclass gets removed before the release (https://docs.travis-ci.com/user/deployment/releases/#troubleshooting-git-submodules). So I added `git submodule update --init --recursive` to `before_deploy` to pull reclass again. I am not 100% sure this will work or not Is there is a way to test this without doing actual release?